### PR TITLE
Updated minimum Atom version to 1.30

### LINKED ISSUE #314: Minimum Atom version too low
I teach a coding course and have 3 students using Ubuntu (two on 18.04, one on 18.10). I also use Ubuntu. We all upgraded to `linter-rubocop@2.2.4`, and stopped seeing any output from `linter-rubocop`. No errors in the console. Downgrading to 2.2.3 fixed it for all of us. Not sure how to go about troubleshooting, let me know if you need any more info.

### PR DESCRIPTION
**Pull request by vzamanillo**
Fixes #314, activation hooks support by grammar scope was implemented on Atom v1.30

### CODE REVIEW

**Comment by vzamanillo** [comment]
Looks like Rubocop release 0.68.0 does not include the cop name in the message anymore.

**Comment by Arcanemagus** [comment]
Your commit message needs to follow the required convention as well:
https://travis-ci.org/AtomLinter/linter-rubocop/jobs/526116217#L525

Note that if you run `npm i` on `master` it should install a `git` hook that checks that automatically for you before it gets committed 😉.

File: src/index.js [hunk 1]
```diff
@@ -73,7 +68,7 @@ const forwardRubocopToLinter = (version, {
 
   const linterMessage = {
     url,
-    excerpt: hasCopName ? excerpt : `${copName}: ${excerpt}`,
+    excerpt: `${copName}: ${excerpt}`,
```
Comment (vzamanillo) [inline src/index.js]: This change breaks the compatibility with versions between 0.52.0 and 0.68.0 :/

**Comment by Arcanemagus** [comment]
Mind splitting f795954 off into a separate PR? I prefer to keep PR's down to a single purpose so it's easier to come back to them later and see right away what they did without having to dig into the diff to realize that this _also_ modified the behavior for `rubocop@<0.68.0`.

**Comment by vzamanillo** [comment]
#322

**Pull request merged**